Sources/PrimerSDK/Classes/Core/Payment Services/ApplePayService.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ApplePayService.swift
+//  PrimerSDK
+//
+//  Created by Evangelos Pittas on 16/4/21.
+//
+
+import Foundation
+
+
+protocol ApplePayServiceProtocol {
+    func tokenize(instrument: PaymentInstrument, completion: @escaping (Result<PaymentMethodToken, Error>) -> Void)
+    func fetchConfig(_ completion: @escaping (Error?) -> Void)
```

Comment:
[Q]
This function is called `fetchConfiguration` at the API level. Which convention you want to use? :)

Reply:
Well.. you're right. But the other services have it like `fetchConfig`, so I kept the one from the other services. To tell you the truth, I can rename all of those, they are internal and won't break anything for anyone.

Reply:
I leave it up to you to decide :) maybe we can have a separate PR for this internal renaming?

Reply:
Yeah, actually I'm going to do it alongside with the other rename.